src/formatter/Cli.hx

```diff
@@ -161,6 +159,16 @@ class Cli {
 			if (content == null) {
 				Sys.exit(-1);
 			}
+			if (paths.length != 1) {
+				Sys.println(content);
+				Sys.exit(3);
```

Comment:
Hm.. shouldn't there be an actual error message printed to stderr for these failure cases? Seems easy to miss that anything is wrong otherwise if you don't check the exit code.